CHANGELOG.md

```diff
@@ -1,3 +1,16 @@
+## v0.28.0 [2021-08-04]
+
+_What's new?_
+
+- New tables added
+  - [aws_emr_cluster_metric_isidle](https://hub.steampipe.io/plugins/turbot/aws/tables/aws_emr_cluster_metric_isidle) ([#570](https://github.com/turbot/steampipe-plugin-aws/pull/570))
+
+_Bug fixes_
+
+- Fixed: Exception handling for organization trail arn in `aws_cloud_trail` ([#578](https://github.com/turbot/steampipe-plugin-aws/pull/578))
```

Comment:
```suggestion
- Fixed: `aws_cloudtrail_trail` table is now smarter when hydrating data for shadow trails (global and organization) ([#578](https://github.com/turbot/steampipe-plugin-aws/pull/578))
```

The table name in the original note was also correct, sometimes the commit message has the wrong table name (which is another issue), but the release note author should review commit messages and fix any mistakes in the notes.